field-sustainability-helper/src/index.css

```diff
@@ -1,3 +1,12 @@
 body {
   margin: 0;
 }
+
+.progressBar {
+  height: 20px;
+  width: 160px;
```

Comment:
```suggestion
```

I agree here - let's just remove the height and width rules, because it looks fuzzy as-is:

![image](https://user-images.githubusercontent.com/209355/174891903-a5366d0d-c695-484c-ae15-7d2bc1c3bafd.png)